environments/sbox.tfvars

```diff
@@ -21,4 +20,38 @@ frontends = [
     certificate_name = "STAR-sandbox-platform-hmcts-net"
     disabled_rules   = {}
   },
+  {
+    name             = "idam-web-public"
+    custom_domain    = "idam-web-public-aks.sandbox.platform.hmcts.net"
+    backend_domain   = ["firewall-sbox-int-palo-sbox.uksouth.cloudapp.azure.com"]
+    certificate_name = "STAR-sandbox-platform-hmcts-net"
+    disabled_rules   = {}
+    global_exclusions = [
+      {
+        match_variable = "QueryStringArgNames",
+        operator       = "StartsWith",
+        selector       = "password"
+      },
+      {
+        match_variable = "QueryStringArgNames",
+        operator       = "Equals",
+        selector       = "label"
+      },
+      {
+        match_variable = "QueryStringArgNames",
```

Comment:
won't these go in the request body?
https://www.terraform.io/docs/providers/azurerm/r/front_door_firewall_policy.html#selector
options are:

> selector - (Optional) Match against a specific key if the match_variable is QueryString, PostArgs, RequestHeader or Cookies.

Reply:
Please note the match variables from FD https://docs.microsoft.com/en-gb/azure/web-application-firewall/ag/custom-waf-rules-overview#match-variable-required are not the same as https://www.terraform.io/docs/providers/azurerm/r/front_door_firewall_policy.html#match_variable-1 as this is a `managed_rule.exclusion.match_variable` rather than a `custom_rule.match_condition.match_variable` https://www.terraform.io/docs/providers/azurerm/r/front_door_firewall_policy.html#match_variable